src/main/java/seedu/address/storage/JsonAdaptedLesson.java

```diff
@@ -101,6 +103,9 @@ public Lesson toModelType() throws IllegalValueException {
         }
         final Date modelDate = new Date(StringUtil.stripLeadingZeroes(strippedDate));
         final Date modelEndDate = new Date(StringUtil.stripLeadingZeroes(strippedEndDate));
+        if (modelDate.isAfter(modelEndDate)) {
```

Comment:
Is end date same as date meant to be valid?

Reply:
Yes, you can start and end the lesson on the same date.

Reply:
Behaviour is consistent with calendar apps out there, but true that it doesn't make sense to have a single date recurring lesson. I don't think it's an issue though